core/src/main/java/nl/nn/adapterframework/jdbc/MessageStoreListener.java

```diff
@@ -101,11 +101,7 @@
 	public void configure() throws ConfigurationException {
 		super.configure();
 		if (sessionKeys != null) {
-			sessionKeysList = new ArrayList<>();
-			StringTokenizer stringTokenizer = new StringTokenizer(sessionKeys, ",");
-			while (stringTokenizer.hasMoreElements()) {
-				sessionKeysList.add(stringTokenizer.nextToken());
-			}
+			extractSessionKeyList();
```

Comment:
Code extracted to a public method for testability and being able to avoid calling `super.configure()`, which crashes without having more configuration and a database -- that is more hassle than I want to add for the tests in the 7.8 branch, this close to release.